vendor/engines/projects/spec/controllers/projects/projects_controller_spec.rb

```diff
@@ -0,0 +1,115 @@
+require "rails_helper"
+require_relative "../../projects_spec_helper"
+
+RSpec.describe Projects::ProjectsController, type: :controller do
```

Comment:
I don't know how you feel about it, but I don't feel strongly about needing to namespace the engine specs in the filenames. I.e. I would have no problem if this were `vendor/engines/projects/spec/controllers/projects_controller_spec.rb`. The `app` directory needs it for autoloading to work properly, but specs don't.